thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/wrapper/AnomalyDetectorWrapper.java

```diff
@@ -151,7 +152,7 @@ public DetectionPipelineResult run() throws Exception {
     // 1. get the last time stamp for the time series.
     // 2. to calculate current values and  baseline values for the anomalies detected
     // 3. anomaly detection current and baseline time series value
-    if( this.cachingPeriodLookback > 0) {
+    if (this.cachingPeriodLookback >= 0) {
```

Comment:
Why >= 0? If it is zero then there is no need to cache.

Reply:
Not necessarily. If it's 0, i.e. the cache look back is 0, which means it only caches current time series. If it's negative, then it means the cache is turned off.